Python/compile.c

```diff
@@ -950,42 +952,46 @@ codegen_addop_j(instr_sequence *seq, location loc,
     return _PyInstructionSequence_Addop(seq, opcode, target.id, loc);
 }
 
-#define RETURN_IF_ERROR_IN_SCOPE(C, CALL) { \
-    if ((CALL) < 0) { \
-        compiler_exit_scope((C)); \
-        return ERROR; \
-    } \
-}
+#define RETURN_IF_ERROR_IN_SCOPE(C, CALL) \
+    do { \
+        if ((CALL) < 0) { \
+            compiler_exit_scope((C)); \
+            return ERROR; \
+        } \
+    } while(0)
 
 #define ADDOP(C, LOC, OP) \
     RETURN_IF_ERROR(codegen_addop_noarg(INSTR_SEQUENCE(C), (OP), (LOC)))
 
-#define ADDOP_IN_SCOPE(C, LOC, OP) RETURN_IF_ERROR_IN_SCOPE((C), codegen_addop_noarg(INSTR_SEQUENCE(C), (OP), (LOC)))
+#define ADDOP_IN_SCOPE(C, LOC, OP) \
+    RETURN_IF_ERROR_IN_SCOPE((C), codegen_addop_noarg(INSTR_SEQUENCE(C), (OP), (LOC)))
```

Comment:
Could you avoid stylistic changes to code you're not changing?

```suggestion
#define ADDOP_IN_SCOPE(C, LOC, OP) RETURN_IF_ERROR_IN_SCOPE((C), codegen_addop_noarg(INSTR_SEQUENCE(C), (OP), (LOC)))
```

Reply:
I personally think it's fine to change that one because I was checking whether there was a semi-colon at the end of the macro or not (the `VISIT` macro did have one and I spotted it easily because it was not at the end of my screen). 

I can do a separate PR but again, I feel that's a bit too much (IIRC, this is the only macro that was missing this new line in the file).

Reply:
You don't need to create a separate PR, if you think it makes it easier to check for the semicolon it's fine.